html/index.html

```diff
@@ -588,7 +588,27 @@ <h6>Comment lines</h6>
               </div>
             </div>
           </div>
-          <div class="col-12 col-md-4">
+          <div class="col-12 col-md-2">
+            <div class="row">
+              <div class="col">
+                <h5 class="my-3">Repository topics</h5>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col">
+                <div class="mb-3">
+                  <input type="text"
+                         name="topics"
+                         id="search-topics"
+                         class="form-control mb-3"
+                         placeholder="Has topic"
+                         aria-label="Repository Topic"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-md-2">
```

Comment:
As stated in the meeting, move this to the top of the form, near the license and label filter. The order should be 'license', 'topic', 'label'